mlflow/tracking/_model_registry/utils.py

```diff
@@ -81,6 +82,12 @@ def _get_registry_uri_from_spark_session():
     session = _get_active_spark_session()
     if session is None:
         return None
+
+    dbr_version = get_databricks_runtime_version()
+    if dbr_version and dbr_version.startswith("client."):
+        # Connected to Serverless
+        return "databricks-uc"
```

Comment:
Shouldn't this be used when `spark.mlflow.modelRegistryUri` is missing?

Reply:
@BenWilson2 suggests to only provide default value in serverless: https://databricks.slack.com/archives/C022KUWGVQS/p1713194084260239?thread_ts=1712956996.629289&cid=C022KUWGVQS

Reply:
got it.